Dockerfile

```diff
@@ -40,3 +45,19 @@ RUN SECRET_KEY=none django-admin compress
 
 # Run application
 CMD gunicorn wagtailio.wsgi:application
+
+
+FROM backend AS dev
+
+# Install Heroku CLI
+RUN curl https://cli-assets.heroku.com/install-ubuntu.sh | sh
+
+# Install AWS CLI
+RUN curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "/tmp/awscli-bundle.zip"
+RUN unzip /tmp/awscli-bundle.zip -d /tmp
+RUN /tmp/awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
+
+# Install development Python requirements.
+COPY requirements.txt /
+COPY requirements-dev.txt /
```

Comment:
Nitpick: you could save a layer here by doing `COPY requirements.txt requirements-dev.txt /` - hardly the end of the world though :smile:

Reply:
Done, thanks!